ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -602,7 +602,6 @@ Loader {
                         width: 40
                         height: 40
                         name: root.senderIcon || ""
-                        assetSettings.isImage: root.isDiscordMessage || root.senderIcon.startsWith("data")
```

Comment:
Discord special case dropped?

Reply:
Yes, seems not needed. I tested discord messages and contact images are still there.

Reply:
Cool